### ISSUE TITLE
Initial postgresql support available

### ISSUE THREAD

**Issue by nesiax** [effgarces/BookedScheduler#10]
Hi, this is not a issue per se, but I developed a BookedScheduler version for postgresql and is already available for download at: https://github.com/nesiax/PGBookedScheduler in case you want to test it.

**Comment by effgarces** [comment]
Awesome work, I don't usually work pgsql, but I really should., it seems much more stable.
If we could merge this support in a single version it would be  great.  I need 48h days

**Comment by nesiax** [comment]
Hi,

Unfortunately it is not easy to merge the work into a single one.

Booked Scheduler don't use a ORM, and there is a file at lib/Database/Commands/Queries.php with almost all sql queries for the database that must be adjusted for MySQL and PostgreSQL. I modified all the queries and adapt them for PGSQL.

But that is not the only file that created sql statements, there are a lot more at: Domain/Access/ and lib/Database/ so not easy as it seems.

Also I found some error on the application that were not noticeable while working with MySQL mostly because the app turn off error logging, but they appear in the 'Reports' page and the 'Groups' page.

I will keep an eye on what you add to BookedScheduler in order to add them to PGBookedScheduler and keep them in sync.

Give PostgreSQL a try, you will love it !
